production/scrapers/maryland_youth.R

```diff
@@ -10,9 +10,9 @@ maryland_youth_restruct <- function(x){
     md_tab <- magick::image_read_pdf(x)
     
```

Comment:
General comment about this file now that we have added youth to the possible options of  jurisdictions do we want to change this scrapers jurisdiction from state to youth?

Reply:
You have picked on a weird thing about the youth scrapers. There's basically a two-tier system for crosswalking, where the state-specific ones work like all other scrapers, and the manual youth scraper has `jurisdiction` as youth for a specific crosswalking behavior. 

For the *automated* ones, this is how cross-walking works: 
- Jurisdiction = state or county, and scraper returns state name (here, "MD") 
- `clean_facility_name` merges on xwalk_name_raw, state, and jurisdiction like normal

For the *manual* youth scraper, on the other hand: 
- Jurisdiction = youth, and scraper returns state as "Not Available" 
- `clean_facility_name` treats them like immigration/federal facilities, and merges on xwalk_name_raw only (see code here: https://github.com/uclalawcovid19behindbars/behindbarstools/pull/110/files#diff-d0fc1e2d4051eda1c57d14a37125ca76e8f4d4a8a1073b313fd1e1f2ec4aefe3R78) 

Totally open to changing this since I realize it's confusing, this was the simplest way I could think to do it!